requirements.txt

```diff
@@ -10,6 +10,7 @@ numpy
 pytest-qt
 confluent-kafka
 ess-streaming-data-types >= 0.9.5
+babel
```

Comment:
I don't see that we are using this anywhere here or am I just blind? :)

Reply:
Ah, that's true. Its from my attempt at fixing/improving parsing of text field values. I will remove it.
```suggestion
```